src/main/javascript/src/components/ListParticipants.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="mt-4" v-if="listOfParticipants.length > 0">
+    <v-row class="mx-3">
+      <v-row align="center" class="mx-3">
+        <v-checkbox
+          on-icon="$checkboxIndeterminate"
+          color="primary"
+          :value="tempSelectedInAGroup.length > 0"
+        />
+        {{ tempSelectedInAGroup.length }} Selected
+      </v-row>
+      <!-- Dropdown Menu -->
+      <v-menu offset-y>
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn color="primary" v-bind="attrs" v-on="on">MOVE TO</v-btn>
+        </template>
+        <v-list>
+          <v-list-item-group>
+            <template v-for="(option, index) in moveToOptions">
+              <v-list-item v-if="index.toString() !== selectedOption" :key="option">
+                <v-list-item-action
+                  v-on:click="
+                    moveToHandlerComponent(option, tempSelectedInAGroup)
+                  "
+                >
+                  <v-list-item-content>
+                    <v-list-item-title>{{ option }}</v-list-item-title>
+                  </v-list-item-content>
+                </v-list-item-action>
+              </v-list-item>
+            </template>
+          </v-list-item-group>
+        </v-list>
+      </v-menu>
+    </v-row>
+
+    <!-- List of available participants -->
+    <v-list class="mt-5" outlined rounded>
+      <v-list-item-group v-model="tempSelectedInAGroup" multiple>
+        <template v-for="(participant, index) in listOfParticipants">
+          <v-list-item :key="participant.userId" :value="participant">
+            <v-list-item-action>
+              <v-checkbox color="primary"></v-checkbox>
+            </v-list-item-action>
+
+            <v-list-item-content>
+              <v-list-item-title>
+                {{ participant.user.displayName }}
+              </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+          <v-divider
+            v-if="index !== listOfParticipants.length - 1"
+            class="mx-4"
+            :key="participant.userId"
+          />
+        </template>
+      </v-list-item-group>
+    </v-list>
+  </div>
+</template>
+<script>
+export default {
+  name: "ListParticipants",
+  props: [
+    "listOfParticipants",
+    "moveToOptions",
+    "moveToHandler",
+    "selectedOption",
+  ],
+  data() {
+    return {
+      tempSelectedInAGroup: [],
+    };
+  },
+  methods: {
+    moveToHandlerComponent(option, tempSelectedInAGroup) {
+      const _this = this;
```

Comment:
You shouldn't need to bind this scope to _this in this context. You can remove this line and change the following references of `_this` to `this`

Reply:
Understood. Refactoring.